test/cross-browser/report.js

```diff
@@ -0,0 +1,262 @@
+/*!
+ * test/launch.js
+ */
+
+/* -----------------------------------------------------------------------------
+ * dependencies
+ * ---------------------------------------------------------------------------*/
+
+// Core
+var spawn = require('child_process').spawn,
+    path  = require('path');
+
+/* -----------------------------------------------------------------------------
+ * setup
+ * ---------------------------------------------------------------------------*/
+
+var webdriver = require('selenium-webdriver');
+var By = webdriver.By;
+var logging = webdriver.logging;
+var until = webdriver.until;
+var promise = webdriver.promise;
+
+// hack required to set should on Driver derived promises
+Object.defineProperty(webdriver.promise.Promise.prototype, 'should', {
+    get: Object.prototype.__lookupGetter__('should')
+});
```

Comment:
This code is specific to `chai-as-promised` and can be removed if `chai-as-promised` is not going to be utilized.